test/generators/ModelGenerators.scala

```diff
@@ -210,6 +210,15 @@ trait ModelGenerators {
       } yield Address(addressLine1, addressLine2, postalCode, country)
     }
 
+  implicit lazy val arbitraryPostalCodeAddress: Arbitrary[PostalCodeAddress] =
+    Arbitrary {
+      for {
+        addressLine1 <- stringsWithMaxLength(AddressLine1.streetNumberLength, Gen.alphaNumChar)
```

Comment:
The messages indicate that this could probably be `Gen.alphaChar`. Minor change though as this should have no effect on tests.